hivemind/p2p/p2p_daemon_bindings/control.py

```diff
@@ -65,17 +80,17 @@ def __init__(
         self.daemon_connector = daemon_connector
         self.handlers: Dict[str, StreamHandler] = {}
 
-    async def _handler(self, reader: asyncio.StreamReader, writer: asyncio.StreamWriter):
-        pb_stream_info = p2pd_pb.StreamInfo()  # type: ignore
-        await read_pbmsg_safe(reader, pb_stream_info)
-        stream_info = StreamInfo.from_protobuf(pb_stream_info)
-        try:
-            handler = self.handlers[stream_info.proto]
-        except KeyError as e:
-            # should never enter here... daemon should reject the stream for us.
-            writer.close()
-            raise DispatchFailure(e)
-        await handler(stream_info, reader, writer)
+        # persistent connection readers & writers
+        self._pers_conn_open: bool = False
```

Comment:
nit: In my opinion, the `pers` part is unclear.

```suggestion
        self._persistent_conn_open: bool = False
```